File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.java
class Solution {
    public String solution(String s) {
        if(s.length() % 2 == 0){
            char[] ch = new char[2];
                ch[0] = s.charAt(s.length()/2-1);
                ch[1] = s.charAt((s.length()/2));
            String answer = new String(ch);
            return answer;
        }
        else{
            char[] ch = new char[1];
            ch[0] = s.charAt(s.length()/2);
            String answer = new String(ch);
            return answer;
        }
        

    }
}

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.cs
public class Solution {
    public string solution(string s) {
       if(s.Length % 2 == 0){
            char[] ch = new char[2];
                ch[0] = s[s.Length/2-1];
                ch[1] = s[s.Length/2];
            return new string(ch);
        }
        else{
            char[] ch = new char[1];
            ch[0] = s[s.Length/2];
            return new string(ch);
        }
    }
}